router/core/router_metrics.go

```diff
@@ -14,47 +14,40 @@ import (
 type RouterMetrics interface {
 	StartOperation(logger *zap.Logger, requestContentLength int64, sliceAttr []attribute.KeyValue, inFlightAddOption otelmetric.AddOption) *OperationMetrics
 	ExportSchemaUsageInfo(operationContext *operationContext, statusCode int, hasError bool, exportSynchronous bool)
-	GqlMetricsExporter() *graphqlmetrics.Exporter
+	ExportSchemaUsageInfoPrometheus(operationContext *operationContext, statusCode int, hasError bool, exportSynchronous bool)
+	GqlMetricsExporter() *graphqlmetrics.GraphQLMetricsExporter
```

Comment:
```suggestion
	GQLMetricsExporter() *graphqlmetrics.GraphQLMetricsExporter
```

https://go.dev/wiki/CodeReviewComments#initialisms